### ISSUE TITLE
Link synchronization not working

### ISSUE THREAD

**Issue by penguin0616** [Falconne/ImprovedWorkbenches#98]
I have 3 stonecutters tables.
I have linked a bill between the 3 of them to make marble blocks, which is set to make 30x (Do X times)

However, when a stone block is made at a table, the bill number only decreases at that table.
The remaining number of blocks to do does not remain consistent across tables.

However, lowering or increasing the amount of blocks to make with the - and + buttons resynchronize the bill amounts, but the issue still persists.

I am not sure if this intended behavior or not.

**Comment by Falconne** [comment]
Hmm, I would consider that expected behaviour. If you just wanted 30 blocks you'd want to create jobs set to "Make until 30" (either linked or not). In my mind, 3 "Make until X" jobs implies make 3 x X items regardless of link state.

**Comment by penguin0616** [comment]
Make until X means that it would take into account the items I already have, which is why I was avoiding that option. Using "Make until 30" means that any unrelated consumption/use of what I already have made means that the pawns will make more than I want them to, when I try to schedule any other tasks (ie construction) at the same time.

**Comment by Falconne** [comment]
Makes sense. I don't know when I'll next have time to work on mods again if ever, but will leave this request here, maybe someone interested can make a pull request. If you're able to, feel free to have a go.
